### ISSUE TITLE
Error with Azure SQL Database External Source

### ISSUE THREAD

**Issue by ChRocheFR** [microsoft/sqlworkshops-sql2019workshop#3]
While trying to do  the module 8 activity, I have an error whil trying to connect to an Azure SQL Database from my local SQL Server 2019 (Dev Edition with PolyBase).
Tried several syntax to create external source and this one should work:
```
CREATE EXTERNAL DATA SOURCE AzureSQLDatabase
WITH ( 
LOCATION = 'sqlserver://<someServer>.database.windows.net',
CREDENTIAL = AzureSQLDatabaseCredentials 
)
GO
```
But I only have this message: 
`Le fournisseur OLE DB "MSOLEDBSQL" du serveur lié "(null)" a retourné le message "Impossible de générer le contexte SSPI".
Msg -2146893042, Niveau 16, État 1, Ligne 3
SQL Server Network Interfaces: Aucune information d’identification n’est disponible dans le package de sécurité
`
I have no problem to connect with SSMS and credential is fine

**Comment by rgward** [comment]
Thank you for reporting this issue.  I have a few thoughts:

1. Please ensure you have installed the latest cumulative update for SQL 2019. There were some Polybase issues for GA. 
2. Can you connect with SSMS to your Azure SQL database logical server using the same credentials you used to create the database scoped credential?

**Comment by ChRocheFR** [comment]
I will be honest, I moved on since. PolyBase was just a test and I uninstall it after these errors. 
For your thoughts: 1. the installation of SQL 2019 was just fresh, so I think it was up-to-date. 2. I do not remember if I used my admin account or another one, but I'm pretty confident that a connection with SSMS would have worked.

Anyway, I would like to thank you for your workshop, it was really interesting!
